modules/apigee_edge_teams/src/Entity/Controller/TeamAppController.php

```diff
@@ -20,6 +20,7 @@
 
 namespace Drupal\apigee_edge_teams\Entity\Controller;
 
+use Apigee\Edge\Api\ApigeeX\Controller\AppByOwnerControllerInterface as XAppByOwnerControllerInterface;
```

Comment:
can we call this AppByOwnerControllerInterface and drop the X

Reply:
We already have `AppByOwnerControllerInterface as EdgeAppByOwnerControllerInterface;` (line 25) for Edge, to prevent ambiguity we have Apigee X file name as XAppByOwnerControllerInterface